src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -4359,39 +4359,39 @@
      * <p>For example:</p>
      * <pre>
      * return switch (day) {
-     *     case SAT, SUN {@code ->} "Weekend";
-     *     default {@code ->} "Working day";
+     *     case SAT, SUN -&gt; "Weekend";
+     *     default -&gt; "Working day";
      * };
      * </pre>
      * <p>parses as:</p>
      * <pre>
-     *  LITERAL_RETURN (return)
-     *   |--EXPR
-     *   |   `--LITERAL_SWITCH (switch)
-     *   |       |--LPAREN (()
-     *   |       |--EXPR
-     *   |       |   `--IDENT (day)
-     *   |       |--RPAREN ())
-     *   |       |--LCURLY ({)
-     *   |       |--SWITCH_RULE
-     *   |       |   |--LITERAL_CASE (case)
-     *   |       |   |   |--EXPR
-     *   |       |   |   |   `--IDENT (SAT)
-     *   |       |   |   |--COMMA (,)
-     *   |       |   |   `--EXPR
-     *   |       |   |       `--IDENT (SUN)
-     *   |       |   |--LAMBDA {@code ->}
-     *   |       |   |--EXPR
-     *   |       |   |   `--STRING_LITERAL ("Weekend")
-     *   |       |   `--SEMI (;)
-     *   |       |--SWITCH_RULE
-     *   |       |   |--LITERAL_DEFAULT (default)
-     *   |       |   |--LAMBDA {@code ->}
-     *   |       |   |--EXPR
-     *   |       |   |   `--STRING_LITERAL ("Working day")
-     *   |       |   `--SEMI (;)
-     *   |       `--RCURLY (})
-     *   `--SEMI (;)
+     *  LITERAL_RETURN -&gt; return
```

Comment:
lets put `--` in front of `LITERAL_RETURN`, and shift right following lines

Reply:
Thanks for the review. I have made the requested change.

Reply:
@timurt , as we agreed in other PR, we do not need such leading indentation.